packages/renderer/src/views/CreateNewAddressbookEntry.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <div class=" bg-background ml-4 max-w-lg rounded-lg  my-10">
+    <header class=" max-w-lg pb-10">
+      <h1 class="  text-left text-4xl font-bold text-on_background">
+        Create a new Addressbook Entry
+      </h1>        
+    </header>
+    <form class="w-100">
+      <main>
+        <!----- Label ---->
+        <FormInput
+          id="label"
+          v-model="addedlabel"
+          div-class="w-80"
+          label="Label"
+        />
+        <!----- Description ----->
```

Comment:
could we stick to two dashes for the HTML comments. More seem to confuse the GitHub syntax higlighting